tests/rptest/clients/rpk.py

```diff
@@ -67,9 +68,9 @@ def __eq__(self, other):
 class RpkGroupPartition(typing.NamedTuple):
     topic: str
     partition: int
-    current_offset: int
-    log_end_offset: int
-    lag: int
+    current_offset: Union[int, str]
```

Comment:
I think the only string tolerated here is "-", right?  It would be neater to use an Optional[int] and convert the "-" into a None, rather than callers having to know how to handle a string here

Reply:
done.